src/com/owncloud/android/ui/errorhandling/ErrorMessageAdapter.java

```diff
@@ -443,7 +443,7 @@ private static String getGenericErrorMessageForOperation(RemoteOperation operati
                 ) {
             message = res.getString(R.string.share_link_file_error);
 
-        } else if (operation instanceof UnshareOperation) {
+        } else if (operation instanceof RemoveShareOperation) {
             message = res.getString(R.string.unshare_link_file_error);
```

Comment:
We prefer not renaming strings. When we do that, they are handled by Transifex as if a string was removed and other was added, so that we lose the translations and the same string needs to be translated again.